cmd/keeper/keeper.go

```diff
@@ -1754,3 +1756,45 @@ func keeper(c *cobra.Command, args []string) {
 
 	lockFile.Close()
 }
+
+func (p *PostgresKeeper) generateHBA(cd *cluster.ClusterData, role common.Role, userHba []string) []string {
+
+	// Dynamically generate hba auth from clusterData
+	var hbaBase []string
+
+	if cluster.PgHBAAccessAll == *cd.Cluster.Spec.PgHBAAccess {
```

Comment:
Use cd.Cluster.DefSpec().PgHBAAccess or you'll miss the default values when not provided

Reply:
Done !